chainio/clients/wallet/wallet.go

```diff
@@ -16,4 +17,5 @@ type TxID = string
 type Wallet interface {
 	SendTransaction(ctx context.Context, tx *types.Transaction) (TxID, error)
 	GetTransactionReceipt(ctx context.Context, txID TxID) (*types.Receipt, error)
+	SenderAddress(ctx context.Context) (common.Address, error)
```

Comment:
Why is this needed? Can you add comment explaining its usecase. Doesn't seem like its used anywhere in this PR, unless I missed it.

Reply:
I wanted to use this when you pass in the sender address when you create the txMgr

Reply:
yep ok just saw this in avs-sync! Ok that makes sense. Can you just add a comment here explaining that this is needed to create a txMgr that wraps the wallet, since the wallet hides the signer key/address, whereas the txMgr needs to know the from address (forget why though lol)

Reply:
done